src/components/aws/costBreakdown/ChartComponent.js

```diff
@@ -180,6 +181,12 @@ export class Header extends Component {
       </div>
     ) : null);
 
+    const tooltip = ((this.props.type === "pie" || this.props.type === "bar") ? (
+      <div className="inline-block">
+        <Tooltip icon={<i className="fa fa-info-circle btn btn-default"/>} tooltip="To hide an item, click on it. Double-click to select only one item." placement="left"/>
```

Comment:
Not clear if you are talking about the item on the chart or on the legend. Please change it to something like : 
"To hide/display a data series please click on its name in the legend. Double-click will display only this item"